front_end/core/host/RNPerfMetrics.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright 2024 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Copyright (c) Meta Platforms, Inc. and affiliates.
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
```

Comment:
I think you need to add this new file (and other) to https://github.com/facebookexperimental/rn-chrome-devtools-frontend/blob/8c5e07a93900cdfc87174c3b82908ff1d75cf79f/scripts/eslint_rules/lib/check_license_header.js#L69-L72 in order to get correct headers

Reply:
Ah. Some eslint rule was looking for `addEventListener` and expects a second param. That crashed so the license linter didn't run either